lib/llvmopencl/SubCfgFormation.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * This file is part of hipSYCL, a SYCL implementation based on CUDA/HIP
+ *
```

Comment:
Please point to the origin (in github) so we can then more easily synch fixes from upstream.